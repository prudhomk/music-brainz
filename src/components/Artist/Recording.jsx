import React from 'react';
import PropTypes from 'prop-types';

export default function Recording({ title }) {
  return (
    <>
      <h1>{title}</h1>
    </>
  );
}

Recording.propTypes = {
  title: PropTypes.string.isRequired,
};
