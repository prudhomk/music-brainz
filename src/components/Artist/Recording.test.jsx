import React from 'react';
import { render } from '@testing-library/react';
import Recording from './Recording';


describe('Recording component snapshot', () => {

  it('render Recording', () => {
    const { asFragment } = render(
      <Recording />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
