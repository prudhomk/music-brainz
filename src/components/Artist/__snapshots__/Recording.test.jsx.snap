// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recording component snapshot render Recording 1`] = `
<DocumentFragment>
  <h1 />
</DocumentFragment>
`;
